src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -248,6 +248,19 @@ class Validator_30 : public AbstractValidator {
                        const uint32_t dilationW,
                        OvGnaType inPrecision,
                        bool exception = true) const override;
+
+    bool ValidateDwsc(const std::string& name,
+                      const uint32_t inHeight,
```

Comment:
`in_height`
`in_width`
and so on

Reply:
It's in line with the whole file, I don't see a point in changing naming convention for a single piece of code in a whole file.